### PR TITLE
Update wording for multi-invite web flow (ENGDOCS-847)

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Updated wording for new multi-invite web flow.
https://deploy-preview-15213--docsdocker.netlify.app/docker-hub/members/#invite-members-via-docker-id-or-email-address

### Related issues (optional)

ENGDOCS-847

### CODE REVIEW

File: docker-hub/members.md [hunk 1]
```diff
@@ -18,11 +18,12 @@ Use the following steps to invite members to your organization via Docker ID or
 1. Go to **Organizations** in [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}, and select your organization.
 2. In the **Members** tab, select **Invite Member**.
 3. Select **Emails Or Docker IDs**.
-4. Enter the invitee’s Docker ID or email address to invite, and select a team from the drop-down list.
+4. Enter up to 1000 Docker IDs or email addresses to invite. Separate multiple entries using a space or comma.
```
Comment (jerae-duffin) [inline docker-hub/members.md]: Can we say: Enter the applicable Docker IDs or email addresses you want to invite, up to the max of 1000. Separate the multiple entries using a space or comma.

Comment (craig-osterhout) [reply docker-hub/members.md]: Thank you. Updated in the feedback commit below.

File: docker-hub/members.md [hunk 2]
```diff
@@ -18,11 +18,12 @@ Use the following steps to invite members to your organization via Docker ID or
 1. Go to **Organizations** in [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}, and select your organization.
 2. In the **Members** tab, select **Invite Member**.
 3. Select **Emails Or Docker IDs**.
-4. Enter the invitee’s Docker ID or email address to invite, and select a team from the drop-down list.
+4. Enter the Docker IDs or email addresses that you want to invite, up to a maximum of 1000. After each entry, press `Enter`to enter a new entry, or copy and paste multiple entries that are separated by a comma, space, or newline.
```
Comment (jerae-duffin) [inline docker-hub/members.md:21]: LGTM, add a space after the ` to **add** a new entry

Comment (craig-osterhout) [reply docker-hub/members.md]: Thank you, fixed in the commit below.

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Pull request merged**